homeassistant/components/device_tracker/__init__.py

```diff
@@ -167,8 +167,9 @@ def async_setup_platform(p_type, p_config, disc_info=None):
                 raise HomeAssistantError("Invalid device_tracker platform.")
 
             if scanner:
-                yield from async_setup_scanner_platform(
-                    hass, p_config, scanner, tracker.async_see)
+                hass.async_add_job(
```

Comment:
We should not do this because we can't guarantee that the whole system is setup when we fire HOMEASSISTANT_START.